pack/set.go

```diff
@@ -40,7 +41,7 @@ var (
 func NewSet(db string) (*Set, error) {
 	pd := filepath.Join(db, "pack")
 
-	paths, err := filepath.Glob(filepath.Join(pd, "*.pack"))
+	paths, err := filepath.Glob(filepath.Join(escapeGlobPattern(pd), "*.pack"))
```

Comment:
Do you think that this change is test-able in gitobj? If not, I think either the integration test(s) in LFS, or a "unit" test that creates a repository might be appropriate.

Reply:
I don't think this is testable in `gitobj`.  We don't have any test code that calls `NewSet`, only some of the more internal functions.

As for an integration test in Git LFS, I have already written such a test, have already verified it works, and will include it when I bump the version of `gitobj` in the main repository.